maven-failsafe-plugin/pom.xml

```diff
@@ -266,12 +266,12 @@
                                     <postBuildHookScript>verify</postBuildHookScript>
                                     <settingsFile>src/it/settings.xml</settingsFile>
                                     <skipInvocation>${skipTests}</skipInvocation>
-                                    <streamLogs>true</streamLogs>
```

Comment:
why are we changing this line ?

Reply:
Because the info logs do not help. And debug logs cannot be read in the main logs.
Several days ago we fixed logs archives and so these logs of this module can be downloaded as a zip.